state-chain/pallets/cf-ingress-egress/src/lib.rs

```diff
@@ -425,9 +425,10 @@ pub mod pallet {
 		/// representation.
 		type AddressConverter: AddressConverter;
 
-		/// Pallet responsible for managing Liquidity Providers.
-		type LpBalance: LpBalanceApi<AccountId = Self::AccountId>
-			+ LpDepositHandler<AccountId = Self::AccountId>;
+		/// Pallet responsible for managing account balances.
+		type Balance: BalanceApi<AccountId = Self::AccountId>;
+
+		type LpDepositHandler: LpDepositHandler<AccountId = Self::AccountId>;
```

Comment:
I think we can remove this and use the BalanceApi instead. The only problem is that we fire an event in the LP pallet which could be a breaking change for product.

Reply:
It doesn't seem they use it.